relationships/synthesis/EXT_INFRA-KUBERNETES_CONTAINER-to-EXT-SERVICE.yml

```diff
@@ -0,0 +1,36 @@
+relationships:
+  - name: extInfraK8sContainerEXTService
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Metric" ]
+      - attribute: entity.type
+        anyOf: [ "SERVICE" ]
+    relationship:
+      expires: P75M
+      relationshipType: HOSTS
+      source:
+        buildGuid:
+          account:
+            lookup: true
+          domain:
+            value: INFRA
+          type:
+            value: CONTAINER
+          identifier:
+            fragments:
```

Comment:
can you point us to a thread where this guid building was discussed? to verify is correct.

Reply:
I have tagged you in the slack thread. Thanks.